### PR TITLE
Fix/find resources within organization and models validations

### PR DESCRIPTION
**Pull request by fbalarini**
### Brief Description
* Find resources within organization from logged user.
* Remove some validations from controllers and adding it to the corresponding model.
### Related card or ticket
https://github.com/orgs/Pis-moove-it/projects/7#card-14013219
https://github.com/orgs/Pis-moove-it/projects/7#card-14062372
### Additional Details
I had to fix and update some tests after doing this changes.

### CODE REVIEW

File: app/controllers/pockets_controller.rb [hunk 1]
```diff
@@ -4,40 +4,48 @@ def index
...
   def edit_weight
-    pocket = Pocket.find(params[:id])
-    return render_error(1, 'Missing weight') if check_invalid_entry('weight')
-    return render_error(1, 'Unweighead pocket') if pocket.state == 'Unweighed'
-    return render_error(1, 'Negative weight') if negative_weight?
-
-    pocket.update!(weight: params['weight'])
-    render json: pocket
+    return render_error(1, 'Missing weight') if params[:weight].blank?
+    return render_error(1, 'Unweighead pocket') if pocket.Unweighed?
```
Comment (accg14) [inline app/controllers/pockets_controller.rb]: Just a detail, missing t in 'Unweighead'

File: app/models/pocket.rb [hunk 2]
```diff
@@ -5,6 +5,7 @@ class Pocket < ApplicationRecord
   belongs_to :collection
 
   validates :serial_number, presence: true
+  validates :weight, numericality: { greater_than_or_equal_to: 0 }, allow_nil: true
```
Comment (gastonabella) [inline app/models/pocket.rb]: Could a pocket weigh 0?

File: app/models/route.rb [hunk 3]
```diff
@@ -5,7 +5,9 @@ class Route < ApplicationRecord
 
   has_many :collections, dependent: :destroy
 
+  validates :length, numericality: { greater_than_or_equal_to: 0 }, allow_nil: true
```
Comment (gastonabella) [inline app/models/route.rb]: Same here

File: app/models/pocket.rb (hunk 2 above)
Comment (fbalarini) [reply app/models/pocket.rb]: good catch

**Review by gastonabella — APPROVED** [review]

File: spec/models/pocket_spec.rb [hunk 4]
```diff
@@ -27,5 +28,15 @@
       pocket.serial_number = nil
       expect(pocket).not_to be_valid
     end
+
+    it 'is not valid with an invalid wieght' do
+      pocket.weight = 0
+      expect(pocket).not_to be_valid
+    end
+
+    it 'is not valid with an invalid wieght' do
```
Comment (GonzaNunezCano) [inline spec/models/pocket_spec.rb:37]: Just one detail, weight is misspelled

**Pull request merged**